resources/xsd/fastdds_profiles.xsd

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles"
-           targetNamespace="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles"
+           xmlns="http://www.eprosima.com/XMLSchemas/fastdds_Profiles"
+           targetNamespace="http://www.eprosima.com/XMLSchemas/fastdds_Profiles"
```

Comment:
Maybe these lines could be removed. If at some point the schema is hosted in the company webpage, these lines can be added again. What do you think, @EduPonz?

Reply:
Based on [this](https://www.w3schools.com/xml/schema_schema.asp) platform, those tags indicate the project default namespace and that the elements defined by this schema come from that namespace, respectively.
I would keep them both, but removing the `XMLSchemas` subsection.

```suggestion
           xmlns="http://www.eprosima.com/fastdds_Profiles"
           targetNamespace="http://www.eprosima.com/fastdds_Profiles"
```

Reply:
I think we can point to the repository's schema instead (the raw link). In fact, part of the release commit could be setting them to a specific version.

regarding the `targetNamespace` I don't know what it is so it's difficult for me to comment on it

Reply:
Not being sure how this works I have just left as it is and I have only update to the fastdds name.